src/main/java/frc/robot/constants/RobotConstants.java

```diff
@@ -48,6 +49,12 @@
         public double TARGET_NOT_FOUND_WAIT_TIME;
     }
 
+    public static class ClimbConstants {
+        public double CLIMB_THRESHOLD_LIMIT;
+        public double CLIMB_CURRENT_LIMIT;
+        public double CLIMB_TIMEOUT;
```

Comment:
CLIMB_CURRENT_TIMEOUT
change climb motors and hook motor (talon srx and spark max)

Reply:
What do you mean? i did this already in an earier code review